packages/cashier/src/Components/Form/account-transfer-form.jsx

```diff
@@ -241,6 +241,10 @@ const AccountTransferForm = ({
     const is_mt_transfer = selected_to.is_mt || selected_from.is_mt;
     const is_dxtrade_transfer = selected_to.is_dxtrade || selected_from.is_dxtrade;
 
+    if (selected_from.value.startsWith('VRTC') || selected_from.value.startsWith('DXD')) {
+        location.reload();
+    }
+
```

Comment:
Why do we need to reload the page for these specific accounts?

Reply:
To not to see demo accounts here. You named it as specific.
When the page is refresh, MobX return correct values which we need to use in transfer page

The page should refresh not very often, just for this special case

Reply:
This is already handled in account transfer store. We are mounting the component on switch of accounts that's why there is no need to reload the page. I suggest to take a look at the `onMountAccountTransfer` function in the store and check if the account list is updated on switch of accounts.

Reply:
Please check the new way. Do you meant as here?